docs/components/Address.md

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 1
+---
+
+# Address
+
+Display an address (or ENS) plus a copy address icon utility. If the address has an ENS with avatar associated, it will show this avatar, if not it will show a blockie image for that address.
+
+By default, clicking on the address sends you to blockexplorer details for that address.
+
+![Address Example](/img/Address.png)
+
+## Import
+
+```ts
+import { Address } from "~~/components/scaffold-eth";
+```
+
+## Usage
+
+```ts
+const [selectedAddress, setSelectedAddress] = useState("");
+
+<Address address={selectedAddress} />;
+```
```

Comment:
`NITPICK` : 

I was wondering what if just put in a hardcoded address : 

```tsx
<Address address="0x34aA3F359A9D614239015126635CE7732c18fDF3" />
```

So that people can just copy and paste in their code? 

or if we want instead of using aribitary `useState` we could make it more real life : 

```tsx
const { address: connectedAddress } = useAccount();
<Address address={connectedAddress} />
```

but yeah even the current looks fine to me!

Reply:
I like the `<Address address="0x34aA3F359A9D614239015126635CE7732c18fDF3" />` very simple and newbie-friendly. Just pushed the tweak 🔥

I like the real use case also, but maybe we can add it in the future Recipes 🙌